README.md

```diff
@@ -68,72 +136,23 @@ Example command for DataflowRunner:
 
 ```bash
 python -m gcp_variant_transforms.vcf_to_bq \
-  --input_pattern gs://bucket/vcfs/vcffile.vcf \
-  --output_table projectname:bigquerydataset.tablename  \
-  --project projectname \
-  --staging_location gs://bucket/staging \
-  --temp_location gs://bucket/temp \
-  --job_name vcf-to-bq \
+  --input_pattern gs://my-bucket/vcffiles/*.vcf \
+  --output_table my-project:my-bigquery-dataset.my-bigquery-table \
```

Comment:
I think the dataset and table actually can't have "-" in them. Might be good to change them to "_" even though they're just dummy values.

Reply:
Good point! Done.